rental.rb

```diff
@@ -0,0 +1,11 @@
+class Rental
+  def initialize(date, person, book)
```

Comment:
- [ ] Great job. please, kindly ensure Getters and Setters for @date, @book and @person instance variables should be present. (`attr_accessor`)

Reply:
i have implement this now, i didn't  realise it wasn't added before my first submission. Thank you